.github/workflows/ci.yml

```
@@ -26,71 +26,68 @@ jobs:
-        cd backend
-        pip install -r requirements.txt
-
-    - name: Run tests
-      run: |
-        cd backend
-        python -m pytest tests/ -v
-      env:
-        DATABASE_URL: postgresql://postgres:password@localhost:5432/daily_stock_test
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.8)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)